src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user1 = new UserService();
```

Comment:
make this service a class level field, use a better name, like userService, naming with numbers is a bad practice